AST-1560 - Changelog missing for Hierarchy missing from Product Categories in the Frontend (#4152)
brainstormforce/astra: changelog.txt

- Fix: The Menu get disappear on the Android mobile device when you select Header Type is set as Dropdown.
- Fix: Elementor Popup – Youtube video fullscreen not working when the cart element is added.
- Fix: Submenus are misaligned with old Astra header menu for specific cases.

> - Fix: WooCommerce - In the Side bar, Product Categories hierarchy is missing in the Frontend.

Reviewer: ```suggestion
- Fix: WooCommerce - Product Categories widget's hierarchy layout is inappropriate on the Frontend.
```

Author: @imnavanath - Done
